### ISSUE TITLE
Incorrect E127 for multi-line with statement in block after expression

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#336]
In GitLab by @snoack on Jan 11, 2020, 09:45

*Please describe how you installed Flake8*

`pip install flake8` in *virtualenv* via *tox*.

*Please provide the exact, unmodified output of `flake8 --bug-report`*

```
{
  "dependencies": [
    {
      "dependency": "entrypoints",
      "version": "0.3"
    }
  ],
  "platform": {
    "python_implementation": "CPython",
    "python_version": "3.7.3",
    "system": "Linux"
  },
  "plugins": [
    {
      "is_local": false,
      "plugin": "mccabe",
      "version": "0.6.1"
    },
    {
      "is_local": false,
      "plugin": "pycodestyle",
      "version": "2.5.0"
    },
    {
      "is_local": false,
      "plugin": "pyflakes",
      "version": "2.1.1"
    }
  ],
  "version": "3.7.9"
}
```

*Please describe the problem or feature*

Flake8 is complaining about how the `with`-statement is wrapped here, incorrectly reporting an E127:

```python
if 1:
    foo()
    with open(filename1) as f1, \
         open(filename2) as f2:
        pass
```

Most interestingly, if I either remove the surrounding block (i.e. `if 1:`) or the preceded expression (i.e. `foo()`), no E127 is reported.

**Comment by asottile** [comment]
In GitLab by @asottile on Jan 11, 2020, 09:55

flake8 does not implement any lint checks of its own it is merely a framework -- I believe you're looking for [pycodestyle](https://github.com/pycqa/pycodestyle) -- please search that issue tracker first I'm pretty sure this is a duplicate there

**Comment by asottile** [comment]
In GitLab by @asottile on Jan 11, 2020, 09:55

closed
